Source/Android/app/build.gradle

```diff
@@ -104,6 +104,14 @@ android {
             minifyEnabled true
             shrinkResources true
         }
+
+        vr {
+            signingConfig signingConfigs.debug
+            matchingFallbacks = ['release']
+            applicationIdSuffix ".vr"
+            versionNameSuffix '-vr'
+            jniDebuggable false
```

Comment:
Why `false` for `jniDebuggable`? We have it set to `true` normally, and it seems like the documentation says it does nothing nowadays.

Reply:
I hoped to get release builds performance during the development. But as it is ignored I will set it to `true`.